packages/core/src/plugin/type-gql-plugin-test/generate.ts

```diff
@@ -0,0 +1,17 @@
+import { NestFactory } from '@nestjs/core';
+import { GraphQLSchemaBuilderModule, GraphQLSchemaFactory } from '@nestjs/graphql';
+import fs from 'fs';
+import { printSchema } from 'graphql';
+
+import { CarResolver } from './api/car.resolver';
+
+async function generateSchema() {
+    const app = await NestFactory.create(GraphQLSchemaBuilderModule);
+    await app.init();
+
+    const gqlSchemaFactory = app.get(GraphQLSchemaFactory);
+    const schema = await gqlSchemaFactory.create([CarResolver]);
+    fs.writeFileSync(__dirname + '/schema.graphql', printSchema(schema));
+}
+
+generateSchema();
```

Comment:
Example script to generate `schema.graphql` file for plugin in the root of the plugin folder